### PR TITLE
cloud-image-compose: improve package filtering

### PR DESCRIPTION
**Pull request by thrix**
Include -devel packages, because some of the package build only these
(i.e. rust-xattr)

Signed-off-by: Miroslav Vadkerti <mvadkert@redhat.com>

### CODE REVIEW

**Comment by johnbieren** [comment]
lgtm. @joejstuart can you make sure this gets merged once it passes stage? I'll be offline

File: config/Dockerfiles/cloud-image-compose/virt-customize.sh [hunk 1]
```diff
@@ -80,7 +80,7 @@ EOF
 
 virt-copy-in -a ${DOWNLOADED_IMAGE_LOCATION} ${CURRENTDIR}/testrepo/${package} ${CURRENTDIR}/test-${package}.repo /etc/yum.repos.d/
 
-for pkg in $(repoquery --disablerepo=\* --enablerepo=${package} --repofrompath=${package},${rpm_repo} --all | grep -v '\-debug\|\-devel\|.src' | rev | cut -d '-' -f 3- | rev ) ; do
+for pkg in $(repoquery -q --disablerepo=\* --enablerepo=${package} --repofrompath=${package},${rpm_repo} --all | grep -v '\-debuginfo$\|\-debugsource$\|\.src$' | sed 's|-[^-]*-[^-]*$||') ; do
```
Comment (ignatenkobrain) [inline config/Dockerfiles/cloud-image-compose/virt-customize.sh]: actually I just realized that first you need to get the name and only after that grep for debuginfo/debugsource

**Comment by thrix** [comment]
We found out that this should be more correct. Note that this can be tried on on rust-xattr and see if it skips the tests, but pipeline passes.

**Comment by thrix** [comment]
@ignatenkobrain

**Comment by joejstuart** [comment]
[merge]

**Comment by ignatenkobrain** [comment]
@joejstuart when this should be deployed to prod? Just want to test it out.

**Comment by thrix** [comment]
[merge]

**Pull request merged**